index.html

```diff
@@ -62,18 +62,22 @@ <h1 class="u-text--c"><span class="t-highlight">K</span>yuuto Bot</h1>
                 anyone wanting to build their own Open Source collaborative
                 project.
               </p>
+              <br>
+              <a class="btn" href="/docs/">Read the docs ></a>
+              <a class="btn" href="https://github.com/Yuuto-Project/kyuuto/" target="__blank">Get coding ></a> <br>
             </div>
             <iframe
               allowtransparency="true"
               class="landing__dcframe"
               frameborder="0"
               height="500"
               src="https://discordapp.com/widget?id=684392803131850779&theme=dark"
-              width="300"
-            ></iframe>
+              width="300">
+          </iframe>
           </div>
         </div>
       </section>
     </main>
+    <br>
```

Comment:
What is the purpose?

Reply:
To have a little extra space under the page,. Without it, the Discord widget touches the bottom.

Reply:
Fix it with styles